### PR TITLE
Console bugs

### PR DESCRIPTION
**Pull request by slipher**
For the 2nd one, there was a Color which was supposed to be statically initialized to white, but apparently ran before White's constructor in an MSVC build, as all the color text showed up as black.

### CODE REVIEW

File: daemon/src/common/Color.h [hunk 1]
```diff
@@ -352,18 +352,18 @@ enum {
...
-extern Color Blue;
-extern Color Yellow;
-extern Color Orange;
-extern Color Magenta;
-extern Color Cyan;
-extern Color White;
-extern Color LtGrey;
-extern Color MdGrey;
-extern Color LtOrange;
+const Color Black    = { 0.00, 0.00, 0.00, 1.00 };
```
Comment (mbasaglia) [inline daemon/src/common/Color.h]: They could be constexpr

**Comment by DolceTriade** [comment]
Could you make it constexpr and we can merge this?

**Comment by DolceTriade** [comment]
constexpr seems to complicate stuff with MSVC. We'll leave as is for now.

**Comment by mbasaglia** [comment]
Even with our macros?

**Pull request merged**